components/brave_sync/client/bookmark_change_processor.cc

```diff
@@ -960,8 +1017,8 @@ void BookmarkChangeProcessor::SendUnsynced(
     sync_client_->SendSyncRecords(
       jslib_const::SyncRecordType_BOOKMARKS, records);
     records.clear();
+    sync_client_->ClearOrderMap();
```

Comment:
?

Reply:
Idea is `sync_client_->ClearOrderMap();` - is not required when `sync_client_->SendSyncRecords()` was not invoked. But in anyway this is not a subject of this PR, so reverted.